### ISSUE TITLE
在沒有預訓練的情況下對 RAF-DB 進行評估

### ISSUE THREAD

**Issue by uu156** [zengqunzhao/MA-Net#10]
您好，我在使用您提供的代碼及步驟進行實驗時，在 RAF-DB 上能夠如論文得到 88% 以上的準確率，但在不使用預訓練模型時無法重現論文中 TABLE II 86.34% 的效果，請問您在沒有預訓練的情況下訓練時有做什麼改動嗎？謝謝！

**Comment by zengqunzhao** [comment]
> 您好，我在使用您提供的代碼及步驟進行實驗時，在 RAF-DB 上能夠如論文得到 88% 以上的準確率，但在不使用預訓練模型時無法重現論文中 TABLE II 86.34% 的效果，請問您在沒有預訓練的情況下訓練時有做什麼改動嗎？謝謝！

Hi, if you train from scratch,  you should adjust the hyper-params: e.g., epochs-300, batch-size-128, factor-0.1, adjust-freq-70
Hope it helps！

**Comment by wwyxabc** [comment]
> > 您好，我在使用您提供的代碼及步驟進行實驗時，在 RAF-DB 上能夠如論文得到 88% 以上的準確率，但在不使用預訓練模型時無法重現論文中 TABLE II 86.34% 的效果，請問您在沒有預訓練的情況下訓練時有做什麼改動嗎？謝謝！
> 
> Hi, if you train from scratch, you should adjust the hyper-params: e.g., epochs-300, batch-size-128, factor-0.1, adjust-freq-70 Hope it helps！

请问在有预训练权重的条件下，按照给出代码的设置，每次模型的准确率都不一样，这是什么原因导致的？

**Comment by zengqunzhao** [comment]
> 
 
Hi, if you want to get the same results from multiple training, you should use a fixed "random seed".

Best,
Zengqun

**Comment by wwyxabc** [comment]
Thanks，What is the random seed you set in this work（MAnet）？

**Comment by zengqunzhao** [comment]
> Thanks，What is the random seed you set in this work（MAnet）？

It varied under different environments, the results I provide don't use a fixed seed. You can try different seeds and then calculate mean and std. That's should be more stable.
